packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 43.14.0 - August 30, 2018
+
+### Added
+
+#### salesforcedx-vscode-replay-debugger
+
+- Support telemetry for capturing execution & errors ([PR #599](https://github.com/forcedotcom/salesforcedx-vscode/pull/599))
```

Comment:
@DatGuyJonathan not sure if we should have this replay debugger change here

Reply:
Let's put this here. We added a bullet point about telemetry as well last week when we instrumented the commands.